### DISCUSSION TITLE
Re: Track played history

### DISCUSSION THREAD

**Discussion by letshin** [music-assistant/support#955] [Feature requests and ideas]
Is there a way  to track history of media played, and to resume that file at a later time? Perhaps as an option?

**Comment by OzGav** [comment]
If you click on the queue you will see a tab called PLAYED ITEMS

**Reply by letshin** [reply]
Thanks. However tracking doesn't work. No matter what the player doesn't resume from the marked position and always restarts the track from the beginning.

**Reply by OzGav** [reply]
OK sorry you mean resume in the middle of a track. Not at this stage. I think there has been a similar request elsewhere. If I find it I will link you to it and close this one. If I can't find it we will keep this for future reference.

**Reply by OzGav** [reply]
Can you provie a little more info though. Obviously you can pause the track. Do you want to be able to pause track A at half way, then play track B then when track A is played again it will start where you left off?

**Reply by letshin** [reply]
That would be ideal. But I guess at the core it would be nice to be able to stop listening to a track/audiobook (say I finished breakfast and am about to start work), and then resume the same podcast at lunch break without having to restart the book. What normally happens is that I would go elsewhere etc and come back and then the position is lost because for whatever reason - eg wife decides to play some music from spotify everything is sorta gone.

**Comment by OzGav** [comment]
Relates to v1
